### PR TITLE
Minor cleanup for odb regression paths (python/TCL) + small fix in dependency install

### PR DESCRIPTION
**Pull request by openroadie**
Fixed issues with dependency installation (missing graphviz/doxygen)
Fixed problems that most likely arose from renaming opendb to odb and restructuring the build directory.

### CODE REVIEW

File: etc/DependencyInstaller.sh [hunk 1]
```diff
@@ -120,11 +120,13 @@ _installUbuntuDev() {
         automake \
         autotools-dev \
         bison \
+	doxygen \
         flex \
         clang \
         g++ \
         gcc \
         git \
+	graphviz \
```
Comment (vvbandeira) [inline etc/DependencyInstaller.sh:129]: Why add these packages? Also, if we add for ubuntu we should add for other all the OS.

**Review by vvbandeira — COMMENTED** [review]
You are missing DCO, click on "Details" to get instructions on how to fix.

image

File: etc/DependencyInstaller.sh (hunk 1 above)
Comment (openroadie) [reply etc/DependencyInstaller.sh:129]: These packages were missing and needed to be installed by hand to get the docker container to work. Will fix for other OSes.

**Comment by openroadie** [comment]
@maliberty I saw the following issues:
- If I build a docker container using the base ubuntu image, I had to manually install doxygen and graphviz
- The odb README file was written back when odb was probably called opendb
- The python and tcl regressions were failing to start because the build structure has changed. 

Is it possible that the base docker image you are using already has doxygen/graphviz which are not standard ?

**Comment by maliberty** [comment]
I'm not aware that we have any dependency on those tools in or out of docker.  What specifically goes wrong when they are not installed?

**Comment by rovinski** [comment]
doxygen is optional as far as I know. It was added by request of a user in https://github.com/The-OpenROAD-Project/OpenDB/pull/124 (before merging odb into OR) because they wanted to get a hierarchy tree generated for odb (there isn't any doxygen-specific documentation in odb).

I'm guessing graphviz is a dependency for doxygen? If so then the same applies.

**Comment by maliberty** [comment]
graphviz will be for doxygen.  There is an optional setup for doxygen in odb but it isn't required and should be skipped if not found.

**Comment by openroadie** [comment]
@maliberty @rovinski Thank you for the reviews. I am still figuring quite a few things out. I will remove the doxygen/graphviz changes and see if I can bypass the build section that required this.

Do the rest of the changes look ok (mostly related to opendb being renamed to odb and changes in the way cmake is generating the build hierarchy).

@vvbandeira I will fix the DCO stuff as well.

**Pull request closed without merging**